documentation/bolt.ditamap

```diff
@@ -62,18 +62,13 @@
             </topicmeta>
         </topicref>
     </topicref>
-    <topicref href="modules.md" format="markdown">
-        <topicref href="bolt_installing_modules.md" format="markdown">
-            <topicmeta>
-                <shortdesc>You can use Bolt to install a Forge module.</shortdesc>
-            </topicmeta>
-        </topicref>
-        <topicref href="module_structure.md" format="markdown">
-            <topicmeta>
-                <shortdesc>Use module structure if you use Forge modules with Bolt, or you want to develop your own modules to share tasks and plans.</shortdesc>
-            </topicmeta>
-        </topicref>
-    </topicref>
+    <topichead navtitle="Modules">
+        <topicref href="modules.md" format="markdown"/>
+        <topicref href="managing_modules.md" format="markdown"/>
+        <topicref href="bolt_installing_modules.md" format="markdown"/>
```

Comment:
I'm not sure how I feel about 'managing modules' and 'installing modules' as verbiage...the new module workflow is also for installing modules, there might be better ways of indicating 'install modules the old-fashioned way' and 'install and manage modules the new experimental way'?

Reply:
Yeah, I actually originally had all of this info on the installing page. But It was too difficult to completely incorporate the docs into the existing installing doc because it quickly became very confusing. The verbiage I settled on was Managing modules "in projects" because that gave me some way to distinguish between the old way and the new way. 

Ultimately, I think this managing page is going to be temporary, anyway. Eventually all of the necessary information for installing modules will be on an "installation" page. I'll think about some different verbiage I can use, though.

Reply:
Those are good points: we definitely don't want to deter people from the installation page, 'manage' to makes sense, and this page seems temporary anyway. If I think of something better I'll let you know, but for now I think this is fine.